### DISCUSSION TITLE
How to save filters

### DISCUSSION THREAD

**Discussion by peckanthony** [rappasoft/laravel-livewire-tables#382] [Q&A]
Hi,

I didn't see this in the docs, but can you save filters in the session so you can navigate to another page and return to the table view with the same filters applied?

**Comment by rappasoft** [comment]
Currently, everything is saved in the query string so if you come back all is wiped out. You can prefill the query string and it will re-filter the table on page load though.

**Reply by peckanthony** [reply]
Would this be an interesting feature? I have a self made datatable in my projects and it stores my filters in session and works lovely with Livewire.

Perhaps it's even possible with https://github.com/reinink/remember-query-strings, which restores your query strings when visiting a certain route. But it's used in Inertia, don't know if it would play well with Livewire.

**Reply by rappasoft** [reply]
It's not a bad feature, I like the idea of remembering the query string since it's technically the default behavior. But I think it should also be an option at both the component level and the user level. I.e. the dev can enable it on the component itself, but the user should choose "remember my settings" or something, in case they want a fresh start every time. Currently I don't have a settings panel anywhere. Maybe I should add a cog icon or something to start one of we're going to add user-configurable features like this.
